cmds/rewrite_package.go

```diff
@@ -43,10 +43,9 @@ import (
 	"path/filepath"
 
 	i18n "github.com/maximilien/i18n4go/i18n"
-	goi18n "github.com/nicksnyder/go-i18n/i18n"
 )
 
-var T goi18n.TranslateFunc
+var T i18n.TranslateFunc
```

Comment:
**Callout**:  To avoid disruption the API  `i18n.TranslateFunc` method  looks the same as the `TranslateFunc` from `goi18n` from version 1